Automattic/jetpack
# AI Proofread: Change Breve request endpoint

Fixes #38043

## Proposed changes:

* Removes the API key input
* Uses Jetpack AI's `useAiSuggestions` hook instead
* Fixes an issue where the grading score would say "Infinity" when there is no text other than whitespace

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Enable Breve
* Open the Jetpack sidebar (post sidebar has a known bug)
* Check that there is no input for the OpenAI key anymore
* Check that "fix with AI" is always on if `requireUpgrade` is false
* Check that clicking on "Fix with AI" sends a request through the Jetpack AI hook (you can see the stream in the console with `debug` on)
* Check that the text is fixed as expected

Note: Requests are still not deducted on the frontend, but should change the backend value
